contrib/node/src/python/pants/contrib/node/tasks/javascript_style.py

```diff
@@ -36,15 +36,15 @@ def register_options(cls, register):
     register('--skip', type=bool, fingerprint=True, help='Skip javascriptstyle.')
     register('--fail-slow', type=bool,
              help='Check all targets and present the full list of errors.')
-    register('--javascriptstyle-dir', advanced=True, fingerprint=True,
-             help='Package directory for lint tool.')
+    register('--color', type=bool, default=True, help='Enable or disable color.')
+    register('--default-eslint-version', default='4.15.0', help='Default ESLint version if not configured.')
```

Comment:
It's odd that this option is located in a different location than the other eslint options... it would probably be good to move this to that same eslint subsystem.

Also, should drop the "default" from the option name and update the help: if someone overrides the value of the option, it would no longer be a default. So just "ESLint version to use" probably.

Reply:
That's a good point.